ksql-engine/src/test/java/io/confluent/ksql/util/AvroUtilTest.java

```diff
@@ -138,9 +149,18 @@ public void shouldFailForInvalidResultAvroSchema()
     KsqlTopic resultTopic = new KsqlTopic("testTopic", "testTopic", new KsqlAvroTopicSerDe
         ());
     Schema resultSchema = SerDeUtil.getSchemaFromAvro(ordersAveroSchemaStr);
+
+    OutputNode outputNode = mock(OutputNode.class);
```

Comment:
This looks identical to code above - extract a method?

Reply:
Done!